### ISSUE TITLE
Fehler

### ISSUE THREAD

**Issue by JohannMuenchhagen** [JohannMuenchhagen/Auftragsverwaltung#6]
Bei dem Reiter Aufträge funktioniert die Anpassung des Namens und des Statuses nicht

**Comment by larsmuenchhagen** [comment]
ja, weil das backend nicht vollständig ist. es fehlt das bereitstellen der produkte für einen bestehenden auftrag und die funktionalität des dazugehörigen updates. beim update werden die gesamten daten eines auftrags (details + produkte) an die api gesendet. so lange diese funktionen nicht bereitstehen, kann die speicherfunktion im fe nicht fertiggestellt werden.
